tools/triage/create_triage_tickets.py

```diff
@@ -95,6 +95,12 @@ def main(args):
         required=False,
         help="PAT (personal access token) for accessing Jira",
     )
+    loginArgs.add_argument(
+        "--pull-secret",
+        default=os.environ.get("PULL_SECRET"),
```

Comment:
just to make it clearer from outside:
```suggestion
        "--pull-secret-file",
        default=os.environ.get("PULL_SECRET_FILE"),
```